packages/components/Shimmer/FRNShimmer.podspec

```diff
@@ -1,14 +1,16 @@
 require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
-version = package['version']
 
 Pod::Spec.new do |s|
-  s.name             = 'FluentUI-React-Native-Shimmer'
-  s.version          = "#{version}"
-  s.summary          = 'Fluent UI React Native Shimmer Control'
-  s.homepage         = "https://www.microsoft.com/design/fluent/#/"
-  s.author           = { "Microsoft" => "fluentuinativeowners@microsoft.com"}
+  s.name         = 'FRNShimmer'
+  s.version      = package['version']
```

Comment:
What's the difference between this and what we used to have for version?

Reply:
We used to have a separate line do this:
`version = package['version']`
and then reference it. Not sure why the indirection when we can do it directly do it here. I copied this heavily off of react-native-webview btw, since that's my go-to react-native-community-module to look at